etcdctl/ctlv3/command/error.go

```diff
@@ -38,5 +40,8 @@ func ExitWithError(code int, err error) {
 	if cerr, ok := err.(*client.ClusterError); ok {
 		fmt.Fprintln(os.Stderr, cerr.Detail())
 	}
+	if errors.Is(err, context.Canceled) {
+		os.Exit(0)
```

Comment:
It's not very intuitive that some of the error codes are ignoring the provided 'code'. 

Maybe it should be sorted out in the upper layer ?

Reply:
If by upper layer you mean `newCheckPerfCommand` or `NewCheckPerfCommand`?

Reply:
I think that `attemptCleanup` would be good enough to expose 'cancellation' as os.Exit(0). 
Logging error there would also work.